src/include/MaxxEffect/Control/Direct/MaxxEffect_Revision.h

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Library revision identification
+ ******************************************************************************/
+#ifndef MAXX_EFFECT_REVISION_H
+#define MAXX_EFFECT_REVISION_H
+
+#include "MaxxEffect/MaxxEffect.h"
+#include "MaxxEffect/MaxxStatus.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*******************************************************************************
+ * Get null-terminated revision string.
```

Comment:
SOF uses doxygen for code documentation, this looks like one of the doxygen supported formats, can you confirm ?

Reply:
yes, this is a doxygen annotation